### PR TITLE
Move `RequestedToCapacityRatio` Args defaults to versioned packages

### PR DESCRIPTION
**Pull request by chendave**
Following API standards, defaults for Plugin Args should be added per version.

Signed-off-by: Dave Chen <dave.chen@arm.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by chendave** [comment]
/hold 
waiting for https://github.com/kubernetes/kubernetes/pull/90544

**Comment by chendave** [comment]
/hold cancel

cc @alculquicondor

File: pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go [hunk 1]
```diff
@@ -76,10 +76,6 @@ func NewRequestedToCapacityRatio(plArgs runtime.Object, handle framework.Framewo
 			resourceToWeightMap[v1.ResourceName(resource.Name)] = 1
 		}
 	}
```
Comment (alculquicondor) [inline pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go:78]: Could you add the same validations as the others?

**Review by alculquicondor — COMMENTED** [review]
Totally missed this, thanks!

@pancernik FYI

File: pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go (hunk 1 above)
Comment (chendave) [reply pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go:78]: sure, will cc you then, this is in my TODO list.

**Comment by chendave** [comment]
@alculquicondor  Is it okay for you to comment `lgtm` for this?

**Comment by chendave** [comment]
Courtesy ping @Huang-Wei @ahg-g , for a `lgtm` label.

File: pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go (hunk 1 above)
Comment (alculquicondor) [reply pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go:78]: I think @pancernik can do it in his PRs for #90859

**Pull request merged**